### ISSUE TITLE
Script does not handle multi-line or multi-value insert statements.

### ISSUE THREAD

**Issue by asmyers** [spacetelescope/jwst-apt-tools#1]
APT uses mult-line and multi-value insert statements for several NIRSpec related tables.  AFAICT this script assumes that every insert statement is on a single line with a single set of values.  I've been intending to cleanup the python script I made to do this and submit a patch to but haven't got around to it.

**Comment by asmyers** [comment]
Attached is a script I threw together for this a few months ago.  I've been meaning to clean it up and submit it as a patch, it is extremely ugly code right now.

[tableextract.zip](https://github.com/spacetelescope/jwst-apt-tools/files/1026602/tableextract.zip)

**Comment by JeffValenti** [comment]
Thanks for reporting this issue. Indeed, the script did not handle multi-line, multi-value insert statements. I was not aware that the sql file could contain such constructs. I think I have fixed this issue in commit bb87697cc28d87afb7094e6c0244f0110a79fc11. I tried using the sqlparse package, but the API is so low level that a lot of supporting code is required. Instead, I generalized my existing approach to handle multi-line, multi-value insert statements. The code still makes some assumptions about the format of the sql file. The code now handles the nirspec_msa_metadata table.

In the same commit, I added logic to handle properly strings with embedded commas, using the csv reader included with python. I had to remove the N before unicode strings. Finally, I restructured the code to make it more object-oriented and more idiomatic of python.
